code/gradle.properties

```diff
@@ -6,13 +6,13 @@ org.gradle.configureondemand=false
 moduleProjectName=analytics
 moduleName=analytics
 moduleAARName=analytics-phone-release.aar
-moduleVersion=2.0.0
+moduleVersion=2.0.1
 
 #bourbon-platform-android-module.gradle
 mavenRepoName=AdobeMobileAnalyticsSdk
 mavenRepoDescription=Android Analytics Extension for Adobe Mobile Marketing
 mavenUploadDryRunFlag=false
-mavenCoreVersion=2.0.0
+mavenCoreVersion=2.0.1
 android.useAndroidX=true
 
 #android.enableJetifier=true
```

Comment:
can we remove this comment in this PR https://github.com/adobe/aepsdk-analytics-android/pull/55